squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +131,39 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    await this.server.updatePlayerList;
```

Comment:
This is a method so should have `()` at the end. Either way, not sure it's actually required here?